projects/kubernetes/kubernetes/kubeadm-init.sh

```diff
@@ -11,9 +11,19 @@ else
     kubeadm init --skip-preflight-checks --kubernetes-version @KUBERNETES_VERSION@ $@
 fi
 for i in /etc/kubeadm/kube-system.init/*.yaml ; do
+    n = $(basename "$i")
```

Comment:
There is a typo here which I somehow missed, need to lose the spaces around the `=` to be valid shell.